lib/liveman_web/controllers/plugs/auth.ex

```diff
@@ -0,0 +1,26 @@
+defmodule LivemanWeb.Plugs.Auth do
+  use LivemanWeb, :controller
+
+  import Plug.Conn
+
+  alias LivemanWeb.V1.ErrorView
+
+  def authenticate_user(conn, _) do
```

Comment:
I wonder if we can just use the plug behaviour
```elixir
@behaviour Plug

def init(opts), do: opts

def call(conn, _) do
  ...
```
This way in the router, we can just pipe through like
```elixir
# router.ex
plug LivemanWeb.Plugs.Auth
```
What do you think? @junan 🤔

Reply:
Also, shall we move the `plugs` folder up 1 level so that it's at the `liveman_web` level? 😄 
```markdown
├── liveman_web
│   └── controllers
│   └── plugs
```

Reply:
I checked a bit function plug vs module plug. I found that function plug can be used for simple things and module plug useful for when app grows(like doing heavy initialization). I updated it by using module plug though our app is small.

I moved `plugs` folder one level up also